data/2022-04-01.MrMIME.md

```diff
@@ -13,7 +13,7 @@ The security of communications poses a seemingly never-ending challenge across C
 
 Five years ago, we started to explore an incredible endeavour on how to securely send and receive email. It was my final year in an internship at Cambridge, and the goal was to develop an OCaml library that could _parse_ and _craft_ emails. Thus, Mr. MIME was born. I even gave a [presentation on it at ICFP 2016](https://www.youtube.com/watch?v=kQkRsNEo25k) and introduced Mr MIME in a [previous post](https://tarides.com/blog/2019-09-25-mr-mime-parse-and-generate-emails). Mr MIME also has been selected by the [NGI DAPSI initiative]((https://tarides.com/blog/2022-03-08-secure-virtual-messages-in-a-bottle-with-scop) last year.
```

Comment:
```suggestion
Five years ago, we started to explore an incredible endeavour on how to securely send and receive email. It was my final year in an internship at Cambridge, and the goal was to develop an OCaml library that could _parse_ and _craft_ emails. Thus, Mr. MIME was born. I even gave a [presentation on it at ICFP 2016](https://www.youtube.com/watch?v=kQkRsNEo25k) and introduced Mr. MIME in a [previous post](https://tarides.com/blog/2019-09-25-mr-mime-parse-and-generate-emails). Mr. MIME was also selected by the [NGI DAPSI initiative]((https://tarides.com/blog/2022-03-08-secure-virtual-messages-in-a-bottle-with-scop) last year.
```

Reply:
I'm not sure what happened with this commit (https://github.com/mirage/mirage-www/pull/735/commits/b662fc1edc86a251820debc40057f407194fab77), so I hope I didn't overwrite this file! Please let me know if I did, and I'll put it back up.